.travis.yml

```diff
@@ -3,6 +3,18 @@ language: ruby
 
 sudo: required
 
+# If the change is a markdown file, don't run CI build. Or at a later point
+# we can set a specific job in here (such as a markdown lint)
+before_install:
+- |
+    if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then
+      TRAVIS_COMMIT_RANGE="FETCH_HEAD...$TRAVIS_BRANCH"
+    fi
+    git diff --name-only $TRAVIS_COMMIT_RANGE | grep -qvE '(\.md$)|(^docs)/' || {
```

Comment:
The subdirectory is "doc", not "docs", isn't it?

Reply:
well spotted. I will do a later change to rename this to /docs which is more common then just doc.